include/pisa/util/inverted_index_utils.hpp

```diff
@@ -12,12 +12,12 @@ void emit(std::ostream &os, const uint32_t *vals, size_t n)
 
 void emit(std::ostream &os, uint32_t val) { emit(os, &val, 1); }
 
-
 // sample_fn must be stable, it must return a sorted vector
 template <typename SampleFn>
 void sample_inverted_index(std::string const &input_basename,
                            std::string const &output_basename,
-                           SampleFn &&sample_fn)
+                           SampleFn &&sample_fn,
+                           std::vector<size_t> &dropped_term_ids)
```

Comment:
Any reason we use a `vector` here and an `unordered_set` later? I realize we are pushing terms in here, and we are finding them in the later case, so I am guessing that is why. I suppose this could be an `unordered_set` too though?

Probably does not matter :smile:

Reply:
Yes, you are right. Lets go with `unordered_set` everywhere